src/ledger/grainAllocation.test.js

```diff
@@ -46,24 +46,18 @@ describe("src/ledger/grainAllocation", () => {
     (policyType: PolicyType) => {
       const policy = deepFreeze({policyType, budget: G.ONE});
 
-      describe("it should return an empty allocation when", () => {
-        const emptyAllocation = deepFreeze({
-          policy,
-          receipts: [],
-        });
-
-        it("the budget is zero", () => {
-          const zeroPolicy = {budget: G.ZERO, policyType};
-          const actual = computeAllocation(zeroPolicy, credHistory, new Map());
+      it("it should return an empty allocation when the budget is zero", () => {
+        const zeroPolicy = {budget: G.ZERO, policyType};
+        const actual = computeAllocation(zeroPolicy, credHistory, new Map());
 
-          expect(actual).toEqual({
-            policy: zeroPolicy,
-            receipts: [],
-          });
+        expect(actual).toEqual({
+          policy: zeroPolicy,
+          receipts: [],
         });
-
-        it("all the cred sums to 0", () => {
-          const actual = computeAllocation(
+      });
+      it("should error when the Cred sums to 0", () => {
```

Comment:
Should have a test case for negative cred as well

Reply:
I considered adding it, but it's already enforced in the outer method:
https://github.com/sourcecred/sourcecred/pull/1966/files#diff-b3948ae03dddf0431e5873d71313bdaeR82-R92

And note the helpers getting tested in this test are only reachable via that wrapper method.